tests/unit/defrag.cpp

```diff
@@ -0,0 +1,522 @@
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE DEFRAG library test
+
+#include <boost/test/unit_test.hpp>
+#include <boost/filesystem.hpp>
+
+#include <future>
+
+#include "eblob_wrapper.h"
+
+#include "library/blob.h"
+#include "library/datasort.h"
+#include "library/list.h"
+
+#include "eblob/eblob.hpp"
+
+#include <vector>
+#include <utility>
+#include <random>
+#include <iostream>
+#include <functional>
+
+#include <sysexits.h>
+#include <err.h>
+
+
+void init_datasort_cfg(eblob_wrapper &wrapper, datasort_cfg &cfg) {
+	memset(&cfg, 0, sizeof(cfg));
+	cfg.b = wrapper.get();
+	cfg.log = cfg.b->cfg.log;
+}
+
+
+std::vector<char> generate_random_data(std::mt19937 &gen, std::uniform_int_distribution<unsigned> &dist) {
+	size_t datasize = 2 * (1 << 20); // 1Mib
```

Comment:
2 * (1 << 20) == 1Mib ?

Maybe to use MB? I'm not sure.

Reply:
https://en.wikipedia.org/wiki/Mebibyte

Reply:
Ok, but either comment or value is wrong. Read 1st line of my 1st comment again.